docs/quickstart/csharp.md

```diff
@@ -25,26 +25,25 @@ Using the .NET Core SDK on Windows, OS X, or Linux, you'll need:
 
 * The .NET Core SDK command line tools. 
 * The .NET framework 4.5 (for OS X and Linux, the open source .NET Framework implementation, "Mono", at version 4+, is suitable) 
-* A NuGet executable (to download the Grpc.Tools package that contains protoc and protobuf binaries for code generation)
 * Git (to download the sample code)
 
 On Windows, using Visual Studio, you'll need: 
 
-* .NET Framework 4.5+, Visual Studio 2013 or 2015.
+* .NET Framework 4.5+
+* Visual Studio 2013 or 2015.
 * Git (to download the sample code)
 
 On OS X, using Xamarin Studio, you'll need:
 
-* Mono 4+
-* Xamarin Studio 5.9+
-* A NuGet executable (to download the Grpc.Tools package that contains protoc and protobuf binaries for code generation)
+* Mono 4.4.2+
```

Comment:
let's add "(or Mono 4+ is sufficient if you manually update nuget to version 2.12)"